src/server/license/server/api_server.go

```diff
@@ -327,7 +354,7 @@ func (a *apiServer) UpdateCluster(ctx context.Context, req *lc.UpdateClusterRequ
 
 func (a *apiServer) ListUserClusters(ctx context.Context, req *lc.ListUserClustersRequest) (resp *lc.ListUserClustersResponse, retErr error) {
 	clusters := make([]*lc.UserClusterInfo, 0)
-	if err := a.env.DB.SelectContext(ctx, &clusters, `SELECT id, cluster_deployment_id, user_address, is_enterprise_server FROM license.clusters WHERE is_enterprise_server = false`); err != nil {
+	if err := a.env.DB.SelectContext(ctx, &clusters, `SELECT id, cluster_deployment_id AS ClusterDeploymentId, user_address AS Address, is_enterprise_server AS EnterpriseServer FROM license.clusters WHERE is_enterprise_server = false`); err != nil {
```

Comment:
✅ We should look into generating these as additional tags.  sqlx looks at `db` struct tags.

Reply:
Yeah, that was a feature gogo had, but I don't have any good ideas for getting it back.  gopatch can't add struct tags.  We could write a generator to make these dbXXX structs separately (complete with ToProto or whatever), if we wanted.

We could make sqlx use the "json" tag instead of "db", but that breaks all the existing hand-written structs that use "db:...", and worse, silently changes the behavior of every database query in the system.  When doing big refactors like this reflection is super scary!

Finally, it's usually dumb luck that you can scan into these things anyway.  You cannot read a time in the DB into a timestamppb.Timestamp, for example.  As soon as you hit one of those non-standard types, you have to do this second struct thing.   (gogo let you use a time.Time instead of a types.Timestamp, avoiding this problem occasionally.)